src/Orleans.Runtime/Streams/PersistentStream/PersistentStreamPullingAgent.cs

```diff
@@ -412,7 +412,7 @@ private async Task<bool> ReadFromQueue(QueueId myQueueId, IQueueAdapterReceiver
             if (queueCache != null)
             {
                 IList<IBatchContainer> purgedItems;
-                if (queueCache.TryPurgeFromCache(out purgedItems))
+                 if (queueCache.TryPurgeFromCache(out purgedItems))
```

Comment:
Looks like superfluous whitespace here
```suggestion
                if (queueCache.TryPurgeFromCache(out purgedItems))
```